src/kiota/KiotaPluginCommands.cs

```diff
@@ -37,6 +37,28 @@ internal static Option<List<PluginType>> GetPluginTypeOption(bool isRequired = t
         typeOption.AddValidator(x => KiotaHost.ValidateKnownValues(x, "type", Enum.GetNames<PluginType>()));
         return typeOption;
     }
+
+    internal static Option<PluginAuthType> GetPluginAuthTypeOption(bool isRequired = false)
+    {
+        var authTypeOption = new Option<PluginAuthType>("--auth-type", "The authentication type for the plugin. Should be a valid OpenAPI security scheme.");
+        authTypeOption.AddAlias("-at");
```

Comment:
```suggestion
```

reason: short command options must be single character. The reason for this is that according to the POSIX standard, `-at` is interpreted as 2 options `-a -t`. adding a 2-letter short option may confuse users on posix terminals

Reply:
Alternative could be this as we have similar aliases in the code e.g. `--cn` for `--client-name`
https://learn.microsoft.com/en-us/openapi/kiota/using
```suggestion
        authTypeOption.AddAlias("--at");
```

Reply:
I think we should pick a clear alias. To me 'at' is not very useful since it's also an english word. If I look at a script and see:

```
kiota <other args> --at ApiKey
```

It won't be clear to me what that option does without running `kiota --help`

Reply:
Keeping `--at` until decided otherwise.